data/avgi/avgi side missions.txt

```diff
@@ -0,0 +1,1048 @@
+			`	"Did you meet them?" you ask.`
+			`	"Yes, yes. Several times. Only the angry one was any good at Trefoil, sadly. Speaking of which -" he deftly rearranges the colored stones, arranging them into a neat pattern. Three triangles, one of each color, angled with their topmost points touching at the center of a six-sided board.`
+
+			scene "scene/trefoil"
+
+			`	"Respectful deference. Which color would you like to play as, question? There is no real difference between them, as far as the game plays."`
+			choice
+				`"Red"`
+				`"Green"`
+				`"Blue"`
+			`	"Acknowledgment. Excellent. Explanation. The objective is to remove all of your pieces from the board by moving them to the opposite edge to your own triangle" he explains, demonstrating with a stone, then showing you how to skip it over connected chains of other pieces, moving to their endpoint. "You must balance your moves between freeing your own pieces and blocking the other colors from their own escape.`
+			`	You delicately take a seat on the rail. It's at least only a little worse than a park bench, and you're thankful for the lower gravity on Peripheria.`
+			choice
+				`	"But there's three sides and only two of us."`
+			`	"Approving. An astute observation. The third color can be played by a third person, in some variations, but in our case we take turns moving for it. It - the Third - is an external actor, under our influence yet outside our absolute control, often used as a pawn to block the other player's moves when opportune. Now, would you like to make the first move, question? Players generally either focus on either evacuating their own pieces or moving to block their opponent's path to escape "`
+
+			choice
+				`(Focus on getting your stones out. Move a stone towards your color's escape.)`
+					goto escape1
+				`(Focus on blocking Wisteria. Move a stone towards his triangle.)`
+					goto blockwisteria1
+				`(Focus on blocking the third color. Move a stone towards its triangle.)`
+					goto blockthird1
+
+			label escape1
+			action
+				"avgi: trefoil classical" ++
+			`	"Approving. A good approach. That is the classical move to make at the start of the game, especially for a beginner."`
+				goto next1
+
+			label blockwisteria1
+			action
+				"avgi: trefoil aggressive" ++
+			`	"Amusement. Ah, already moving to block me, I see. That is often an approach taken by more advanced players, but I must warn you, executing it successfully is difficult."`
+				goto next1
+
+			label blockthird1
+			action
+				"avgi: trefoil chaotic" ++
+			`	"Interest. I do not often see players prioritize blocking the Third, especially as their first move. But perhaps you will surprise me."`
+				goto next1
+
+			label next1
+			`	"Now, after each of our turns, we shall move one of the Third's pieces. You should focus on moving them to your advantage, whether that be by blocking my pieces or keeping them out of the way of your own."`
```

Comment:
All of the labels that lead to this have a closing punctuation mark. Rather then adjusting all of those, how about this?
```suggestion
			`	Slightly adjusting his position on the bench, the Avgi continues. "Now, after each of our turns, we shall move one of the Third's pieces. You should focus on moving them to your advantage, whether that be by blocking my pieces or keeping them out of the way of your own."`
```